ext/storage_sources/local_store/local_store.c

```diff
@@ -492,6 +510,14 @@ local_exist(WT_FILE_SYSTEM *file_system, WT_SESSION *session, const char *name,
     local = FS2LOCAL(file_system);
     path = NULL;
 
+    /* If the file exists directly in the file system, it's not yet flushed, and we're done. */
```

Comment:
Is that comment completely correct? I think a file could also exist in the local file system when it is pulled back in and/or not removed after flushing, right? (That code does not exist yet however.) Mostly I want to find out if there are other assumptions around "not yet flushed" and its existence.

Reply:
The model is that any readonly file will be either in the cloud or in the cache directory.  When a flush happens, we'll copy to the cloud and move it to the cache directory (not sure what ordering).  But after a flush, it won't be immediately in the WT home directory.  I think that lines up with what's happening in the metadata, right?  The `file:xxxx.0003.wt` will go away and `object:xxxx.0003.wtobj` will then be there.